### ISSUE TITLE
Wisp whistle object

### ISSUE THREAD

**Issue by Senoj2693** [exult/exult#395]
I ran into a quest problem where I can't talk to any of the Emps about wisps and get the wisp whistle as a result which I would need for a later quest. Can you please add the wisp whistle to the create object list?

**Comment by DominusExult** [comment]
I think it is a frame of shape 693, you can create it via the create object cheat.
It's more interesting why you can't complete the Wisp's quest. What is your problem there?

**Comment by wench** [comment]
I'm surprised that the whistle isn't listed here: https://wiki.ultimacodex.com/wiki/Ultima_VII_Create_Item_list no reason someone should not add it, wikis can be edited by anyone.

I'm currently playing through the game with the SDL3 build almost up to the wisps, will report if I have problems or not.

**Comment by wench** [comment]
Playing through the quest as intended I had no troubles obtaining the whistle (yeo it's a classic Works for me response). I suspect the problem was unintentionally skipping a step. After getting permission from Salamon for Trellek to join you need to ask Saralek for permission again and tell her about wisps when she says no and she will tell you about the whistle, then ask Trellek about Saralek's idea and then you'll get the option to ask trellek about the whistle.

**Comment by wench** [comment]
This issue has me thinking it might be a nice idea id Exult had a way to display detailed debug information about an item so it would be possible to know exactly what shape, frame and quality an object has. Adding text based create item option into cheat screen also seems like a good idea. Shape browser to create items is a bit clunky at time and you can't choose quality or quantity. *adds these feature to an ever expandng todo list of small nice to have features*

**Comment by Senoj2693** [comment]
No. I only used the first one that came up.

**Comment by Senoj2693** [comment]
I checked the other frames and i found one that looks kinda like it. The wisp whistle itself looks like a small brown cylinder.

**Comment by Senoj2693** [comment]
I finally managed to get Trellek to give me a wisp whistle thanks to talking to the fortune teller near Minoc and then talking to Taylor, one of the Monks of Empath Abbey, about wisps. I looked inside my bag and I noticed that the wisp whistle Trellek gave me looks exactly like one of the whistles in the object creator, specifically the one that looks like a brown exclamation point. So it seems that the wisp whistle I needed was already in the object creator this whole time. It just wasn’t given its own name. Idk why the creators didn’t make that distinction.
